### PR TITLE
dcp-717 updating submission guide

### PR DESCRIPTION
**Pull request by amnonkhen**
some updates
dcp-717
ebi-ait/dcp-ingest-central#717

### CODE REVIEW

File: src/assets/guide-markdown/data-requirements.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ The Human Cell Atlas Data Coordination Platform supports scientists in their atl
 
 Data considered for inclusion in the HCA Data Coordination Platform should meet at least one option per category below:
 
-* **Organism:** Human, Mouse, Other*
+* **Organism:** Human, Mouse, Donkey, Other*
```
Comment (gabsie) [inline src/assets/guide-markdown/data-requirements.md:6]: removing Donkey
```suggestion
* **Organism:** Human, Mouse, Other*
```

File: src/assets/guide-markdown/data-requirements.md [hunk 2]
```diff
@@ -46,11 +46,11 @@ There are three main sources of data considered eligible for the Human Cell Atla
 
 ## Data referenced for the Human Cell Atlas Bionetwork Atlases
 
-The data is referenced by the HCA Bionetwork scientific groups as suitable for integration into one of the HCA Bionetwork atlases.
+The data is referenced by the HCA Bionetwork scientific groups as suitable for sdkjhfspoiwe into one of the HCA Bionetwork atlases.
```
Comment (gabsie) [inline src/assets/guide-markdown/data-requirements.md:49]:
```suggestion
The data is referenced by the HCA Bionetwork scientific groups as suitable for integration into one of the HCA Bionetwork atlases.
```

File: src/assets/guide-markdown/data-requirements.md [hunk 3]
```diff
@@ -46,11 +46,11 @@ There are three main sources of data considered eligible for the Human Cell Atla
...
 
 ## Data directly contributed to the HCA Data Coordination Platform
 
-There is a pre-print or official publication referencing data that meets the criteria for published data (see suitability criteria above), or a supporting justification as to why it should be included in the HCA.
+There is a pre-print or official publication referencing data that meets the criteria for published data (see suitability criteria above), or a supporting justification as to why it should be included in the HCAdsfsdf
```
Comment (gabsie) [inline src/assets/guide-markdown/data-requirements.md:53]:
```suggestion
There is a pre-print or official publication referencing data that meets the criteria for published data (see suitability criteria above), or a supporting justification as to why it should be included in the HCA.
```

**Review by gabsie — CHANGES_REQUESTED** [review]
please edit these

**Review by gabsie — APPROVED** [review]
dcp-717 finalised

**Comment by MightyAx** [comment]
@gabsie @amnonkhen,
I'm confused, why was this PR closed?

**Comment by MightyAx** [comment]
Oh I see, just a knowledge sharing on PRs. hope you all learned something!

**Pull request closed without merging**